Makefile

```diff
@@ -1153,6 +1153,9 @@ KVER ?= $(shell uname -r)
 KSRC := /lib/modules/$(KVER)/build
 MODDESTDIR := /lib/modules/$(KVER)/kernel/drivers/net/wireless/
 INSTALL_PREFIX :=
+# Raspbian kernel is with soft-float.
```

Comment:
Doesn't make sense to set this here for one platform type but not others.

Reply:
Perhaps we should simply ditch `CONFIG_PLATFORM_ARM_RPI` and `CONFIG_PLATFORM_I386_PC` . If there's any case where `-mfloat-abi=softfp` (or `-mfloat-abi=hard`) can be applied / is necessary, we can add a "non-platform" config option. (But is it even necessary? Can't it simply be added to (EXTRA_)CFLAGS manually

Reply:
> Can't it simply be added to (EXTRA_)CFLAGS manually if desired?)

@tomty89 In my experience, a lot of people try to build drivers who have no business doing so and then break things badly. There's a better chance of having them select a particular profile or platform, and then having known/good settings with that platform.

I think the problem here is the platform is a Raspberry Pi 0-2, but instead of running Raspian with Raspian settings, it's running Ubuntu with slightly different settings... if I've understood correctly.